homeworks/03-data-structures/src/BinarySearchTree.test.ts

```diff
@@ -0,0 +1,17 @@
+import {BinarySearchTree} from "./BinarySearchTree";
+import {testTree} from "./testData";
+
+describe('BinarySearchTree', () => {
```

Comment:
I would expect tests of inherited methods as well.

Reply:
is it necessary if the test coverage is 100%?